build/generic/build/pdf.js

```diff
@@ -28,8 +28,8 @@ factory((root.pdfjsDistBuildPdf = {}));
   // Use strict in our context only - users might not want it
   'use strict';
 
-var pdfjsVersion = '1.5.224';
-var pdfjsBuild = '4c8e99a0';
+var pdfjsVersion = '1.5.218';
```

Comment:
@nidafarooqui WIs this an intentional change? If so the version here is 1.5.218 but in build/pdf.js version is 1.5.221.

Reply:
yes it was intentional because I needed to push the build for my change to work.